packages/flame/lib/src/components/position_component.dart

```diff
@@ -87,6 +88,9 @@ class PositionComponent extends Component
     if (angle != 0) {
       transform.angle = angle ?? 0;
     }
+
+    transform.angle += angularOffset;
```

Comment:
This should not add the angular offset, since that would rotate the canvas.

Reply:
Removed this rotation.